### DISCUSSION TITLE
Can new results replace old results vs creating a new group/tab?

### DISCUSSION THREAD

**Discussion by user72356** [mtxr/vscode-sqltools#1048] [Q&A] [answered]
Can the query result replace the previous result instead of creating a new tab after every run?

I found the option called "Reuse Tabs" and set that to "connection" but it doesn't appear to work. New result tabs don't get created, but the current result tab also doesn't update with the new result set.

It's quite annoying to have a million tabs open whenever I design and test a query.

**Comment by gjsjohnmurray** [comment, marked as answer]
The `sqltools.results.reuseTabs` is supposed to address this. Issue https://github.com/mtxr/vscode-sqltools/issues/988 is about a problem seen when this is set to `connection`. If that issue doesn't match what you are seeing please open another one.
